sig-contributor-experience/charter.md

```diff
@@ -1,58 +1,93 @@
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or issues, we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Establish policies, standards, and procedures:
+  - routine GitHub management tasks, including but not limited to: org membership, org permissions, repo creation/administration.
+  - "Org Owner" GitHub permissions, and grant/limit these privileges accordingly
+  - bot accounts, service accounts, webooks, and third-party integrations for all communication platforms that we support but most importantly, GitHub.
+- Establishing a "GitHub Administration team" that will oversee the execution of GitHub management tasks: inviting new members to the org, creating repos, executing moderation decisions, auditing permissions.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required
+- Deliver and iterate on Thursday community meeting by delivering a variety of content for a weekly overview into the contributor community.
+- Strategize, build, and execute on scalable mentoring programs for all contributor levels.
+- Create best practices, trainings, policies, and moderation of Kubernetes communication platforms. This includes user guidelines when the contributor is the user of the platform. Moderation is an always on, never off service that we supply with trusted contributors spanning time zones. This includes immediate action when dealing with code of conduct issues in a public space that are defined below:
+    - All mailing lists/googlegroups where we have ownership, including k-dev@. At least one Owner of the mailing list must be a communications subproject OWNER. [link to mailing list section of moderation guidelines]
+    - Zoom via CNCF funding [link to zoom guidelines]
+    - YouTube via the /kubernetescommunity channel [link]
+    - Slack via CNCF funding [link]
+    - discuss.kubernetes.io via CNCF funding [link]
+- Help onboard new contributors and current into the culture, workflow, and CI of our contributor experience through the constant addition, modification and retirement of contributor documentation and programs tailored to onboarding.
+- Perform issue triage and maintain the kubernetes/community repository.  
+- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
+- Assist SIGs/WGs with organizational management operations as laid out in the sig governance doc [link]
+- Plan and execute on face to face Contributor Summit(s) and our own SIG Face to Face meetings as well as others (upon request).
+- Own and execute on other events that are targeted to the Kubernetes contributor such as the Steering Committee Election process.
```

Comment:
You could maybe roll a few things up under this bullet, e.g.

"Own and execute events that are targeted to the Kubernetes contributor community, including:
- The weekly Kubernetes Community meeting
- Contributor Summits
- SIG Contributor Experience F2F meetings
- Steering Committee elections (though we do not own policy creation, see 'out of scope' below)
- Retrospective moderation for other SIGs upon request
- other events upon request and consideration"